pkg/apis/proto/function/v1/mockgen.go

```diff
@@ -1,3 +1,4 @@
 package v1
 
 //go:generate mockgen -destination funcmock/funcmock.go -package funcmock github.com/numaproj/numaflow-go/pkg/apis/proto/function/v1 UserDefinedFunctionClient
+//go:generate mockgen -destination funcmock/reducemock.go -package funcmock github.com/numaproj/numaflow-go/pkg/apis/proto/function/v1 UserDefinedFunction_ReduceFnClient
```

Comment:
We may use one line for both client and stream mock.

Reply:
What do others think?..

Reply:
I didn't know we could do that.
yeah, it makes sense to keep them in one single file.